Gruntfile.js

```diff
@@ -60,7 +60,8 @@ grunt.initConfig({
 		},
 		all: [
 			"<%= jshint.all %>",
-			"!test/deepEqual.js"
+			"!test/deepEqual.js",
+			"!test/propEqual.js"
```

Comment:
I added test/propEqual.js to the exceptions because the tests in both files are almost the same